scio-test/src/main/scala/com/spotify/scio/testing/BigTableMatchers.scala

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2016 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.testing
+
+import java.lang.{Iterable => JIterable}
+import java.util.{Map => JMap}
+
+import com.google.bigtable.v2.Mutation
+import com.google.bigtable.v2.Mutation.MutationCase
+import com.google.protobuf.ByteString
+import com.spotify.scio.values.SCollection
+import org.scalatest.matchers.{MatchResult, Matcher}
+
+import scala.reflect.ClassTag
+
+/**
```

Comment:
Minor: We use old javadoc format, i.e. no extra space before `*`.
https://github.com/spotify/scio/wiki/Style-Guide#documentation-style